### ISSUE TITLE
CSS Pseudo selectors creating a duplicated link announcement on screenreaders

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#5196]
Our accessibility audit founded a duplicated link announcement for elements, like galley `link` buttons, that use `:before` and `:after` pseudo-selectors.

These selectors are creating a gap that results in duplicated announcement of link. We need to review the CSS and the HTML to find a way to prevent it from creating the empty space that such selectors are resulting.

**Comment by israelcefrin** [comment]
PR to fix it (test passed):
https://github.com/pkp/ojs/pull/3528

**Comment by israelcefrin** [comment]
PRs
OJS - https://github.com/pkp/ojs/pull/3528
OPS - https://github.com/pkp/ops/pull/354

**Comment by NateWr** [comment]
Thanks @israelcefrin. There's a unit test error in the OPS pull request. https://app.travis-ci.com/github/pkp/ops/jobs/583277823 I don't think it's related to your changes, but I don't want to merge it in in case we break the build for others.

@asmecher does this test failure look familiar to you? @israelcefrin has only changed CSS so pretty sure it can't be related:

https://app.travis-ci.com/github/pkp/ops/jobs/583277823#L4233-L4241

**Comment by asmecher** [comment]
@NateWr, I think that test failure is just due to an out of date `lib/pkp` submodule; I've started a round of tests on the `main` branch to see if it's resolved. Please go ahead with this one (review and merge) as though the tests were passing in OPS.

**Comment by NateWr** [comment]
Ok, all merged to `main`. :tada: Thanks everyone!
